src/components/elements/default-tab.js

```diff
@@ -1,9 +1,10 @@
 import React, { useState, useEffect } from 'react'
 import PropTypes from 'prop-types'
 import styled, { css } from 'styled-components'
+import { Text } from './typography'
 import { Flex } from 'components/containers'
-import { Text } from 'components/elements'
```

Comment:
isn't it the same thing?

Reply:
When I use it from `components/elements` an issue of `Cannot create styled-component for component: undefined` is raised